src/pages/GiftAidForm/defaultGiftaidFields.json

```diff
@@ -26,7 +26,7 @@
     "name": "firstname",
     "label": "First name",
     "required": true,
-    "pattern": "^[A-Za-z][A-Za-z\\'\\-]*$",
+    "pattern": "^[A-Za-z'-]{1,25}$",
```

Comment:
This would mean the first name can begin with a ' or -. The previous pattern ensures that the first character is always an alphabet.

Reply:
yeah I was thinking about that, but figured that a name could technically begin with a special character like ' or - and we'd be overkilling it with a lot of restrictions. Come to think of it I think it's fine I think as long as we don't allow ppl to just enter spaces. Like what will we say in the error message then if we do allow a ' but not at the beginning of a string?

Btw I didn't get any ESLint issues with escaping characters and it did work when testing. Do you have a different ESLint version/rules?

Reply:
The previous patterns are based on the HMRC patterns provided by Corin. According that pattern, the firstname can only start with alphabets.

Reply:
oh yeah the HMRC pattern. Cool will update.

Reply:
cool